tools/setup/emoji/build_emoji

```diff
@@ -211,3 +211,12 @@ def dump_emojis(cache_path):
 
 if __name__ == "__main__":
     main()
+
+thumbsup = ["thumbs", "up"]
```

Comment:
It's a little unconventional to put code here after the `if __name__ == '__main__'` section.